### DISCUSSION TITLE
How are the modulators configured?

### DISCUSSION THREAD

**Discussion by Woolton** [publicsamples/Oi-Grandad#50] [Q&A] [answered]
There are 12 modulators, only 4 voices, and the first voice has no modulator on it seemingly. I've noticed that mod 1 is on voice 2,3 and 4. I don't see any way of changing what mods route to which, and it's not mentioned in the quick start guide
https://files.catbox.moe/3e5sku.png

**Comment by RickTaylor79** [comment, marked as answer]
Modulators are selected by the menu above a red knob. So looking at the image below, you'd click on the mod source to select a modulator, and turn the mod source to set the amount.

![ogmod](https://github.com/user-attachments/assets/49de8b9e-29af-493e-831e-044195684412)

**Reply by Woolton** [reply]
Thanks, I feel kinda dumb now, I didn't put two and two together because I was just turning the mod amount knob not noticing any change
